gen3/lib/aws.sh

```diff
@@ -485,6 +485,15 @@ account_name = "${GEN3_WORKSPACE//_management-logs/}"
 EOM
       return 0
   fi
+
+  if [[ "$GEN3_WORKSPACE" =~ _sftp$ ]]; then
+      cat - <<EOM
```

Comment:
instead of doing this - launch the thing as `__sftp`, and put a `sample.tfvars` in `aws/sftp/` - https://github.com/uc-cdis/cloud-automation/blob/master/doc/terraform/workon.md#use

ex - this kind of thing will work without adding another rule to `aws.sh`:
```
gen3 workon dcfprod whitelists__sftp
```